### PR TITLE
adds viewport meta tag to storybook frame

### PR DESCRIPTION
**Pull request by kvendrik**
### WHY are these changes introduced?
fixes https://github.com/Shopify/polaris-react/issues/896

### WHAT is this pull request doing?
Adds a viewport meta tag to the `iframe` Storybook loads its previews in.

### How to 🎩
1. Run `yarn dev`.
2. Head over to `localhost:6006` and make sure everything looks as you would expect.
3. Head to `localhost:6006/iframe.html?selectedKind=Playground&selectedStory=Playground` on your phone and make sure the playground shows up in the right scale.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
     "heroku-postbuild": "yarn run copy-polaris-tokens",
     "readme-update-version": "node ./scripts/readme-update-version",
     "version": "yarn run readme-update-version",
-    "storybook": "start-storybook -p 6006 --quiet",
+    "storybook": "start-storybook -p 6006 -c .storybook --quiet",
```
Comment (BPScott) [inline package.json]: Storybook looks in the `.storybook` folder by default so I don't think this config adds any value

Comment (kvendrik) [reply package.json]: That's what I thought too but it doesn't seem to work without it 🤔

**Comment by solonaarmstrong** [comment]
Is there something I can do to help move this along?

**Comment by kvendrik** [comment]
@solonaarmstrong could you 🎩? If it looks good I'll merge this. 🙂

**Comment by solonaarmstrong** [comment]
FYI, I was not able to get `localhost:6006` working on my phone. I had to use the IP address.

I 🎩 the full playground and it isn't doing what I would expect.
img_1311

**Comment by kvendrik** [comment]
@solonaarmstrong make sure you navigate to the frame's contents, it won't work if you're in the StoryBook UI. `192.168.0.104:6006/iframe.html?selectedKind=Playground&selectedStory=Playground`

**Comment by solonaarmstrong** [comment]
Ah! The `iframe.html` didn't take the first two times, but the third time it did.

img_1314

**Comment by kvendrik** [comment]
@solonaarmstrong perfect probably a cache issue, does it work consistently now?

**Comment by solonaarmstrong** [comment]
Yes, refreshing it multiple times continues to work. 🎉

**Comment by kvendrik** [comment]
@solonaarmstrong yes preferably to the ["Testing on mobile or a virtual machine
"](https://github.com/Shopify/polaris-react#testing-on-mobile-or-a-virtual-machine) section of the README. Maybe that's something for a separate PR though?

**Comment by kvendrik** [comment]
Issue here: #1088

**Pull request merged**